packages/fx-core/templates/plugins/resource/bot/bicep/input_param.template.bicep

```diff
@@ -0,0 +1,11 @@
+
+param bot_aadClientId string
```

Comment:
parameter naming style should be same with our resource plugins

Reply:
We can have a discussion on the bicep coding style and fix it accordingly in later PR.

Reply:
Resolve the problem for this PR.